source/index.txt

```diff
@@ -8,6 +8,22 @@ The `MongoDB Connector for Spark
 <https://www.mongodb.com/products/spark-connector>`_ provides
 integration between MongoDB and Apache Spark.
 
+.. note::
+
+   Version 10.x of the MongoDB Connector for Spark is an all-new 
+   connector based on the latest Spark API. Install and migrate to 
+   version 10.x to take advantage of new capabilities, such as tighter 
+   integration with Spark 
+   :ref:`Structured Streaming <spark-structured-streaming>`.
+
+   Version 10.x uses a different namespace, 
```

Comment:
Suggestion:
Version 10.x uses a different namespace, 
   ``com.mongodb.spark.sql.connector.MongoTableProvider``.
   This allows you to use old versions of the MongoDB connector 
   (versions 3.x and earlier) in parallel with version 10.x.

Reply:
That's better, done 👍